package/CHANGELOG

```diff
@@ -18,6 +18,13 @@ The rules for this file:
         
   * 0.15.1
 
+Enhancements
+
+  * All classes derived from 'AnalysisBase' now display a ProgressMeter
+  with 'quiet=False'
```

Comment:
This is correct. You are thinking of `verbose=True` which is the normal way people turn on logging and output. For some reason MDAnalysis has decided in the past to rather go with the word `quiet` and there `False` would say the function can print and log stuff for me,